cmd/repo_add.go

```diff
@@ -0,0 +1,90 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/falcosecurity/falcoctl/cmd/internal/validate"
+	"github.com/falcosecurity/falcoctl/pkg/repo"
+	"github.com/go-playground/validator/v10"
+	"github.com/mitchellh/go-homedir"
+	logger "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"os"
+	"path/filepath"
+)
+
+// Defaults
+const ()
+
+var _ CommandOptions = &RepoAddOptions{}
+
+// RepoAddOption represents the `repo add` command options
+type RepoAddOptions struct {
+	RepoPath string
+	RepoFile string
+}
+
+// AddFlags adds flag to c
+func (o *RepoAddOptions) AddFlags(c *cobra.Command) {
+}
+
+// Validate validates the `repo add` command options
+func (o *RepoAddOptions) Validate(c *cobra.Command, args []string) error {
+	if err := validate.V.Struct(o); err != nil {
+		return err.(validator.ValidationErrors)
+	}
+	return nil
+}
+
+// NewRepoAddOptions instantiates the `search registry` command options
+func NewRepoAddOptions() *RepoAddOptions {
+	return &RepoAddOptions{
+		RepoPath: DefaultRepoPath,
+		RepoFile: DefaultRepoFile,
+	}
+}
+
+func NewRepoAddCmd(options CommandOptions) *cobra.Command {
+	o := options.(*RepoAddOptions)
+
+	cmd := &cobra.Command{
+		Use:                   "add",
+		DisableFlagsInUseLine: true,
+		Short:                 "Adds an artifact repository to the falcotl cache",
+		Long:                  "Search a plugin inside the official Falco registry",
```

Comment:
The text does not match the action purpose. Maybe a leftover?

Reply:
Yes indeed!